content/en/docs/best-practices/channel-naming.md

```diff
@@ -175,9 +198,52 @@ candidate/fast/stable to denote the maturity level of the operator.  Using all
 
 Channel promotion is the notion of moving an operator from one channel to 
 another.  For example, consider the case where you have an operator version 
-1.0.1 which is found in a candidate channel, then you might decide to offer 
+`1.0.1` which is found in a candidate channel, then you might decide to offer 
 support for that version and want to move it to a stable channel.  
 
 Today, channel promotion is achieved by creating a new operator version 
-(1.0.2) that is labeled with the channel(s) you want to promote to (as well 
+(`1.0.2`) that is labeled with the channel(s) you want to promote to (as well 
 as any channels you want to keep it in).
+
+# UPGRADES 
+
+OLM provides a variety of ways to specify updates between operator versions. Before
+continue in the following recommendations, make sure you understand its options by checking [Creating an update graph with OLM][upgrade-graph].
+
+## Recommended upgrade path for patch release (bug fixes)
+
+Within a channel each patch release should be directly upgradeable. Use skip or skipRange
+to ensure its behaviour. 
+
+#### Example
+
+If you have the channel `stable-3.7` where the head of channel is `3.7.10` and then, you will provide a new patch release
+with a bug fix using the Operator bundle version `3.7.11` then, ensure that you configure your new publication to skip all 
+that was published in under the channel `stable-3.7` (e.g. skipRange: `">= 3.7.0 < 3.7.11"` replaces: `3.7.10` ).  
```

Comment:
+1 Ben

The problem is that most readers are likely still using the sqlite imperative workflow, so its very difficult (maybe impossible) to track a moving target from the previous y stream.

This would be very easy to implement in FBC, especially with a veneer that implements all the logic that's described.